bindings/nodejs/lib/types/utils/bridge/utils.ts

```diff
@@ -188,3 +188,10 @@ export type __FaucetMethod__ = {
         address: string;
     };
 };
+
+export interface __OutputIdToUtxoInput__ {
+    name: 'outputIdToUtxoInput';
+    data: {
+        outputId: string;
```

Comment:
Does this need to be changed as well?

Reply:
There is an OutputId type but that's also just a string, so doesn't really make a difference